ActionPlans/Start-MailboxMigrationAnalyzer.ps1

```diff
@@ -1106,7 +1106,7 @@ function Create-CommandToInvoke {
         $CommandFor
     )
 
-    $TheResultantCommand = New-Object PSObject
+    $TheResultantCommand = New-Object PSObject 
```

Comment:
please be careful not to change other files; if you do so please don't add that change to the commit

Reply:
Am sorry I didn't pay attention will do in next commits!